### ISSUE TITLE
损失函数 / 优化器 / 训练迭代器

### ISSUE THREAD

**Issue by any86** [any86/Notes#59]
https://www.w3cschool.cn/tensorflowjs/tensorflowjs-y6kx2q2c.html

损失函数（loss function）：这个损失函数代表了给定多项式和数据的匹配程度。 损失函数值越小，那么这个多项式和数据就跟匹配。
优化器（optimizer）：这个优化器实现了一个算法，它会基于损失函数的输出来修正系数值。所以优化器的目的就是尽可能的减小损失函数的值。
训练迭代器（traing loop）：即它会不断地运行这个优化器来减少损失函数。
